mapbox/dr-ui
# Make `css` optional in Edit and CodeSnippet components

This PR makes the `css` prop optional in the Edit and CodeSnippet components.

Fixes https://github.com/mapbox/dr-ui/issues/273

## How to test

1. Open the Edit test case
2. Test the "No CSS" case to make sure JSFiddle and CodePen open (without CSS) and there are no warning logged to the console.
